src/transport_none.c

```diff
@@ -0,0 +1,15 @@
+/* -*- C -*-
+ *
+ * Copyright (c) 2016 Intel Corporation. All rights reserved.
+ * This software is available to you under the BSD license.
+ *
+ * This file is part of the Sandia OpenSHMEM software package. For license
+ * information, see the LICENSE file in the top level directory of the
+ * distribution.
+ *
+ */
+
+#include "transport_none.h"
+
+void *SHMEMX_CTX_DEFAULT = NULL;
+void *SHMEMX_DOMAIN_DEFAULT = NULL;
```

Comment:
Rename SHMEMX -> SHMEM
Also, delete SHMEMX_DOMAIN_DEFAULT

Reply:
It looks like transport_none is also missing the ```extern void * SHMEM_CTX_DEFAULT;``` declaration.

Reply:
Arg.. committed the wrong file.